src/yadism/coefficient_functions/light/n3lo/xc3ns3p.py

```diff
@@ -69,7 +64,19 @@ def c3nm3a(y, args):
             - dl * dl1 * (39.99 + 5.103 * dl - 16.30 * dl1)
             + 0.0647 * y * dl**4
         )
-        + fl02
+    )
+    return res
+
+
+@nb.njit("f8(f8,f8[:])", cache=True)
+def c3nsv3a(y, args):
```

Comment:
```suggestion
def c3nsv3a(y, args):
    # contributions from flavor class fl02
```
let's be explicit (I know it is written below in code, but we could even drop that to stress this is the associated coefficient)